kit/src/main/java/com/oracle/javafx/scenebuilder/kit/util/PaintConvertUtil.java

```diff
@@ -0,0 +1,145 @@
+package com.oracle.javafx.scenebuilder.kit.util;
+
+import javafx.scene.paint.*;
+
+import java.util.List;
+
+/**
+ * Convert Paint to java code or css code;
+ */
+public class PaintConvertUtil {
+
+    private static final int ROUNDING_FACTOR = 10000;//Use for round to 4 decimal places
+
+    public static String convertPaintToCss(Paint fxPaint) {
+        if (fxPaint == null) {
+            return "";
+        }
+        if (fxPaint instanceof LinearGradient) {
+            LinearGradient paint = (LinearGradient) fxPaint;
+            StringBuilder strBuilder = new StringBuilder("linear-gradient(from ")
+                    .append(lenToStr(paint.getStartX(), paint.isProportional()))
+                    .append(" ").append(lenToStr(paint.getStartY(), paint.isProportional()))
+                    .append(" to ").append(lenToStr(paint.getEndX(), paint.isProportional()))
+                    .append(" ").append(lenToStr(paint.getEndY(), paint.isProportional()))
+                    .append(", ");
+            connectCycleMethodAndStops(strBuilder, paint.getCycleMethod(), paint.getStops());
+            return strBuilder.toString();
+        } else if (fxPaint instanceof RadialGradient) {
+            RadialGradient paint = (RadialGradient) fxPaint;
+            StringBuilder strBuilder = new StringBuilder("radial-gradient(focus-angle ").append(round(paint.getFocusAngle()))
+                    .append("deg, focus-distance ").append(round(paint.getFocusDistance() * 100))
+                    .append("% , center ").append(lenToStr(paint.getCenterX(), paint.isProportional()))
+                    .append(" ").append(lenToStr(paint.getCenterY(), paint.isProportional()))
+                    .append(", radius ").append(lenToStr(paint.getRadius(), paint.isProportional()))
+                    .append(", ");
+            connectCycleMethodAndStops(strBuilder, paint.getCycleMethod(), paint.getStops());
+            return strBuilder.toString();
+        } else if (fxPaint instanceof Color) {
+            return toHex((Color) fxPaint);
+        }
+        return "";
+    }
+
+    public static String convertPaintToJavaCode(Paint fxPaint) {
```

Comment:
Please add some tests to confirm the validity of this method.

Reply:
Thanks, the test class has been added;
com.oracle.javafx.scenebuilder.kit.util.PaintConvertUtilTest